packages/state-importer/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Import states to the commercetools platform",
   "main": "lib/main.js",
   "scripts": {
-    "build": "babel src --out-dir lib"
+    "build": "babel src --out-dir lib --config-file '../../babel.config.js'"
```

Comment:
You could maybe also pass it as an env var fro, the host project. BABEL_CONFIG=„blub“ and use it here via interpolation. I may have seen that somewhere. Not sure it serves a bug benefit however as it never moves etc.